code/game/dna/genes/monkey.dm

```diff
@@ -43,7 +43,11 @@
 	H.set_species(H.species.primitive_form)
 
 	if(H.hud_used)
-		H.hud_used.instantiate()
+		qdel(H.hud_used)
+		H.hud_used = null
+
+	if(H.client)
+		H.hud_used = new /datum/hud/monkey(H, ui_style2icon(H.client.prefs.UI_style), H.client.prefs.UI_style_color, H.client.prefs.UI_style_alpha)
```

Comment:
you really need to make monkey huds a hardcoded part of set_species and human/create_mob_hud